### ISSUE TITLE
[Bug]: Multi-stacked bar chart: Role is set as culture value for chart title, when culture is pass

### ISSUE THREAD

**Issue by scharde** [microsoft/fluentui#24267]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
System:
    OS: Windows 10 10.0.22000
    CPU: (8) x64 Intel(R) Core(TM) i7-10510U CPU @ 1.80GHz
    Memory: 3.00 GB / 15.76 GB
  Browsers:
    Edge: Spartan (44.22000.120.0), Chromium (104.0.1293.47)
    Internet Explorer: 11.0.22000.120
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codesandbox.io/s/multistackedbarroleissue-yd3k1z

## Actual Behavior

Accessibility tool showing error for chart title "ARIA roles used must conform to valid values"

![image](https://user-images.githubusercontent.com/29042635/183421887-ae6e0f54-4741-47b3-a95b-036b6be205fb.png)

## Expected Behavior
The role should not be culture, it should be text or nothing just like other charts. 

![image](https://user-images.githubusercontent.com/29042635/183421037-fe20418d-1e32-4f00-aaf1-beac3fbabfd8.png)

### Requested priority

High

### Products/sites affected

M365AdminUx

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by AtishayMsft** [comment]
This issue is fixed with PR #24319
